src/utils/addresses.ts

```diff
@@ -20,15 +20,6 @@ import { Transaction } from '@alephium/sdk/api/explorer'
 
 import { AddressHash, AddressRedux } from '@/types/addresses'
 import { AddressTransaction, PendingTransaction } from '@/types/transactions'
-import { getRandomLabelColor } from '@/utils/colors'
-
-export const sortAddressList = (addresses: AddressRedux[]): AddressRedux[] =>
-  addresses.sort((a, b) => {
-    // Always keep default address to the top of the list
-    if (a.isDefault) return -1
-    if (b.isDefault) return 1
-    return (b.lastUsed ?? 0) - (a.lastUsed ?? 0)
-  })
```

Comment:
One more 🆒 thing about using Redux, is that our state is always sorted properly, so we can ensure that when we fetch addresses to display to the frontend, they are already sorted out! No need for this utility function anymore 💪

Reply:
So nice.